src/runtime/config/config.org

```diff
@@ -134,7 +134,8 @@ SQLALCHEMY_ECHO = False
 SQLALCHEMY_TRACK_MODIFICATIONS = True
 
 # List of allowed origins for CORS
-ALLOWED_ORIGINS = "['*']"
+#ALLOWED_ORIGINS = "['*']"
+ALLOWED_ORIGINS = ["http://outreach.vlabs.ac.in"]
```

Comment:
Checkout [link](https://flask-cors.corydolphin.com/en/v1.8.0/#flask_cors.cross_origin) to use regular expressions.  The idea is to allow all origins within vlabs.ac.in

Reply:
@travula 

Instead setting up allowed origins in flask configuration, we made same on apache hence this can be ignore

Reply:
Allow * here and write the narrative that CORS settings are done in apache and not at flask